stl/inc/yvals_core.h

```diff
@@ -1112,20 +1110,24 @@
 #define __cpp_lib_endian                   201907L
 #define __cpp_lib_erase_if                 202002L
 #define __cpp_lib_generic_unordered_lookup 201811L
-#define __cpp_lib_int_pow2                 202002L
-#define __cpp_lib_is_constant_evaluated    201811L
-#define __cpp_lib_is_nothrow_convertible   201806L
-#define __cpp_lib_list_remove_return_type  201806L
-#define __cpp_lib_math_constants           201907L
-#define __cpp_lib_remove_cvref             201711L
-#define __cpp_lib_shift                    201806L
-#define __cpp_lib_span                     202002L
-#define __cpp_lib_ssize                    201902L
-#define __cpp_lib_starts_ends_with         201711L
-#define __cpp_lib_to_address               201711L
-#define __cpp_lib_to_array                 201907L
-#define __cpp_lib_type_identity            201806L
-#define __cpp_lib_unwrap_ref               201811L
+
+#ifdef __cpp_lib_bitops // TRANSITION, VSO-1020212
+#define __cpp_lib_int_pow2 202002L
+#endif

```

Comment:
I observe that `<bit>` guards its non-`bit_cast` content with `#ifdef __cpp_lib_bitops // TRANSITION, VSO-1020212` instead of having separate sections for that and `__cpp_lib_int_pow2`. This is fine, but I thought I'd mention it.

Reply:
I think I wrote the guard before __cpp_lib_int_pow2 was a thing.